libs/blockchain/src/blockchain.service.ts

```diff
@@ -55,11 +55,11 @@ export interface IHeaderInfo {
 
 const { NUMBER_OF_NONCE_KEYS_TO_CHECK, NONCE_KEY_EXPIRE_SECONDS, getNonceKey } = NonceConstants;
 
-function getNextPossibleNonceKeys(account: string, currentNonce: string | number): string[] {
+function getNextPossibleNonceKeys(currentNonce: string | number): string[] {
   const keys: string[] = [];
   for (let i = 0; i < NUMBER_OF_NONCE_KEYS_TO_CHECK; i += 1) {
     const key = Number(currentNonce) + i;
-    keys.push(getNonceKey(account, `${key}`));
+    keys.push(getNonceKey(`${key}`));
```

Comment:
Reverted this change from a previous PR; turns out we already include the address in the cache key prefix.